tests/integration/ceph_upgrade_test.go

```diff
@@ -175,13 +175,23 @@ func (s *UpgradeSuite) TestUpgradeToMaster() {
 	require.NotEqual(s.T(), 0, numOSDs)
 
 	//
-	// Upgrade Rook from v1.4 to master
+	// Upgrade Rook from v1.5 to master
 	//
 	logger.Infof("*** UPGRADING ROOK FROM %s to master ***", installer.Version1_5)
 	s.gatherLogs(systemNamespace, "_before_master_upgrade")
 	s.upgradeToMaster()
 
 	s.verifyOperatorImage(installer.VersionMaster)
+
+	// patch the cephblockpool CR due to type change
+	logger.Info("*** PATCHING CEPHBLOCKPOOL ***")
```

Comment:
Per our conversation, this is temporary, right? So we can avoid the type change during upgrade.

Reply:
yes, temporary.